app/controllers/foreman_bootdisk/disks_controller.rb

```diff
@@ -40,6 +40,23 @@ def full_host
       end
     end
 
+    def subnet
+      begin
+        subnet = @disk.try(:subnet)
+        fail unless subnet && subnet.try(:tftp)
```

Comment:
I think we should give clear error message here, otherwise we will only see the generic message ending up with `:`.